packages/sdk/src/utils/getGithubIssueUrl.ts

```diff
@@ -71,7 +73,15 @@ export async function wouldYouLikeToCreateANewIssue(options: IssueOptions) {
       title: options.title ?? '',
       body: issueTemplate(platform, options),
     })
-    await open(url)
+
+    /**
+     * This is a workaround for a quirky `open` behavior.
+     * - `await open(url)` correctly opens the browser, returns the control to Node.js and let it exit the process normally.
+     * - `await open(url, { wait: true })` achieves the same on Windows and WSL. Without `{ wait: true }`, `open` would not
+     *   return the control to the Node.js process, which would result in the prisma cli being stuck.
+     */
+    const shouldOpenWait = isWindows() || isWSL
```

Comment:
Ugh, weird that one is a function and the other is not.

Reply:
Yeah, `is-wsl` returns a boolean directly